src/pages/_app.tsx

```diff
@@ -23,4 +85,52 @@ const MyApp: AppType<{ session: Session | null }> = ({
   )
 }
 
-export default trpc.withTRPC(MyApp)
+const getBaseUrl = () => {
```

Comment:
- [ ] Remove this portion of the code and keep the existing as Ziyuan's work deals with TRPC 9 while ours deals with TRPC 10.